bin/land.js

```diff
@@ -0,0 +1,239 @@
+    process.exit(1);
+  }
+  return parseInt(git.stdout.trim());
+}
+
+async function currentBranch() {
+  let git;
+  try {
+    git = await exec('git rev-parse --abbrev-ref HEAD');
+  } catch (error) {
+    console.error(error);
+    process.exit(1);
+  }
+  return git.stdout.trim();
+}
+
+async function httpsGet(sourceBranch, targetBranch) {
+  const options = {
+    path: `/search/issues?q=repo:${await getRepoName()}+is:pr+is:open+head:${sourceBranch}+base:${targetBranch}`,
+    headers: {
+      'user-agent': 'metarhia-api-landing-tool',
+    },
+    host: 'api.github.com',
+  };
+
+  return new Promise((resolve, reject) => {
+    https
+      .get(options, res => {
+        let data = '';
+        res.setEncoding('utf8');
+
+        res.on('data', chunk => {
+          data += chunk;
+        });
+
+        res.on('end', () => {
+          resolve(JSON.parse(data).items[0].html_url);
+        });
+      })
+      .on('error', err => {
+        reject(err.message);
+      });
+  });
+}
+
+async function getRepoName() {
+  let git,
+    name = 'origin';
+  args.forEach(value => {
+    if (value.startsWith('--remote-name=')) {
+      name = value.split('=')[1];
+    }
+  });
+
+  try {
+    git = await exec(`git remote get-url ${name}`);
+  } catch (error) {
+    console.error(error);
+    process.exit(1);
+  }
+
+  const gitConfig = git.stdout.trim();
+
+  return gitConfig.slice(
+    gitConfig.indexOf(':') + 1,
+    gitConfig.lastIndexOf('.git')
+  );
+}
+
+async function gitLog() {
+  let git;
+  try {
+    git = await exec('git log -1 --pretty=format:%B');
+  } catch (error) {
+    console.error(error);
+    process.exit(1);
+  }
+
+  return git.stdout.trim();
+}
+
+if (args.includes('--help')) {
+  console.log(help);
+  process.exit(0);
+} else if (args.includes('--version')) {
+  console.log(toolVersion);
+  process.exit(0);
+}
+
+const onTargetBranch = () =>
+  childProcess.exec(`git checkout ${targetBranch}`, err => {
+    if (err) {
+      console.error(err);
+      process.exit(1);
+    }
+  });
+
+async function lastCommitHash() {
+  let git;
+  try {
+    git = await exec('git log -1 --pretty=format:%h');
+  } catch (error) {
+    console.error(error);
+    process.exit(1);
+  }
+  return git.stdout;
+}
+
+(async () => {
+  const sourceBranch = await currentBranch();
+
+  if (args.includes('--rebase')) {
+    runGit(['rebase', `origin/${targetBranch}`]);
+  }
+
+  if (args.includes('--autosquash')) {
+    runGit(['rebase', '-i', '--autosquash', `HEAD~${await differCommits()}`]);
+  }
+
+  const commitsCount = await differCommits();
+  const differCommitsHash = (await commitsHash(commitsCount)).split('\n');
+  const commitsHashArr = new Array();
+
+  let prUrl,
+    extendedCommit,
+    urlRequest = false;
+
+  for (const hash of differCommitsHash) {
+    childProcess.execSync(`git checkout ${hash}`);
+
+    const gitLogBody = await gitLog();
+
+    if (!gitLogBody.includes('PR-URL:')) {
+      if (!urlRequest) {
```

Comment:
```suggestion
      if (!prUrl) {
```
?
and then just delete `urlRequest`?